pkg/apis/kops/validation/validation.go

```diff
@@ -1025,8 +1025,10 @@ func validateDockerConfig(config *kops.DockerConfig, fldPath *field.Path) field.
 
 func validateRollingUpdate(rollingUpdate *kops.RollingUpdate, fldpath *field.Path, onMasterInstanceGroup bool) field.ErrorList {
 	allErrs := field.ErrorList{}
+	var err error
+	unavailable := 1
```

Comment:
Nit: `maxUnavailable`?  (though I realize this is pre-existing, so /shrug)

Reply:
I had taken the "max" off of both `MaxUnavailable` and `MaxSurge` here for brevity.